drivers/net/adin1110/adin1110.h

```diff
@@ -38,6 +38,9 @@
 #include "no_os_spi.h"
 #include "no_os_gpio.h"
 #include "no_os_util.h"
+#include "no_os_irq.h"
```

Comment:
this might also have an impact, i didn't check the CI, depending on whether existing projects include .h files one by one or using NO_OS_INC_DIRS

Reply:
This is a left over from a previous implementation and is not needed anymore. I'll remove it.